internal/cli/deployments/setup.go

```diff
@@ -201,7 +201,8 @@ func (opts *SetupOpts) createLocalDeployment(ctx context.Context) error {
 
 func (opts *SetupOpts) configureContainer(ctx context.Context) error {
 	envVars := map[string]string{
-		"TOOL": "ATLASCLI",
+		"TOOL":            "ATLASCLI",
+		"RUNNER_LOG_FILE": "/dev/stdout",
```

Comment:
I think we should only do it if `--debug` was used, people expect this docker container to have mongod logs

Reply:
Updated, I also only added the debug level check before getting the logs